docs/guide/build/runtime-layouts/csv-payments-monolith.md

```
@@ -9,6 +9,10 @@ This page documents the current monolith topology in `examples/csv-payments`.
 - Runtime mapping scenario: `examples/csv-payments/config/runtime-mapping/monolith.yaml`
 - Build script: `examples/csv-payments/build-monolith.sh`
 
+## Source layout clarification
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)